src/game/scenes/levels/Level3.js

```diff
@@ -0,0 +1,37 @@
+import { Pig } from '../../objects/Pig';
+import { Vector } from '../../shared';
+import { levelFrom } from '../LevelScene';
+import { Decoration } from '../../objects';
+
+const Level3 = levelFrom({
+  map: `
+  XXXXXXXXXXXXXXXXXXXXXXXXX
+  XXX.......XX............X
+  XXX........X............X
+  X............XXXXXXXXXXXX
+  X.....XXX....X..........X
+  X.........XX........XX..X
+  X..XX............XX.....X
+  X...XX........X.......XXX
+  X......XXXXXX..........XX
+  X...................X...X
+  X.S........XX......XX...X
+  XXXX......XXX..XX...XXXXX
```

Comment:
Z tego schodka można bezpośrednio wskoczyć na platformę nad nim omijając sporą część poziomu, więc sugeruję go przesunąć w lewo

```suggestion
  X.S.......XX.......XX...X
  XXXX.....XXX...XX...XXXXX
```

Reply:
Jasne, ja nawet tak myślałam, że to może być taki skrót który nie każdy zauważy, ale może lepiej żeby ten ostatni poziom nie miał takich rzeczy.